webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala

```diff
@@ -1824,6 +1823,207 @@ class OntologyV2R2RSpec extends R2RSpec {
       }
     }
 
+    "update the property comment and check if the property is still correctly marked as `isEditable: true`" in {
+      // update label and comment of a property
+      val ontoIri          = SharedOntologyTestDataADM.ANYTHING_ONTOLOGY_IRI_LocalHost
+      val propertyIri      = ontoIri + "#hasOtherNothing"
+      val propertyValueIri = ontoIri + "#hasOtherNothingValue"
```

Comment:
Cannot these 3 be defined once in the entire file and reused instead of dining them in every test?

Reply:
true